esmvaltool/cmorizers/obs/cmorize_obs_esacci_oc.py

```diff
@@ -45,17 +46,43 @@ def _fix_data(cube, var):
 
 def _add_depth_coord(cube):
     """Add depth auxiliary coordinate for CMIP5 standard."""
+    def adjust_dim(dim):
+        if dim == 0:
+            return dim
+        return dim + 1
     if not cube.coords('depth'):
-        depth = 1.
-        depth_coord = iris.coords.AuxCoord(
-            depth,
+        assert len(cube.shape) == 3
```

Comment:
please don't call assertions without providing an exception that gets raised if the condition fails. Here just check for the cube shape and if it's not 3, then raise an exception and terminate the run, eg `if len(cube.shape) != 3: raise ValueError(f"Cube {cube} dimension is not 3")`